jobs/README.md

```
@@ -0,0 +1,52 @@
+AWS Lambda jobs for ADB, deployed with [AWS SAM](https://docs.aws.amazon.com/serverless-application-model/).
+
+## Prerequisites
+
+- Go (preinstalled in the devcontainer)
+- Python + pip (preinstalled in the devcontainer)
+- AWS credentials with deploy permissions for `us-west-2`
+- SSM parameters `smtp_user` and `smtp_pass` set in `us-west-2`
+
+Install Python dependencies (currently just `aws-sam-cli`):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)